scripts/borg.exe.spec

```diff
@@ -38,3 +40,16 @@ exe = EXE(pyz,
           strip=False,
           upx=True,
           console=True )
+
+if False:
```

Comment:
I don't agree, having the explanation what this is about is quite sufficient in such a rarely touched file. No extra flag is needed.